mapper.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var (
-	metricLineRE      = regexp.MustCompile(`^(\*\.|[^*.]+\.)*(\*|[^*.]+)$`)
+	metricLineRE      = regexp.MustCompile(`^(\*\.|[^*.]+\.|\.)*(\*|[^*.]+)$`)
```

Comment:
You are right. Unfortunately, this means anyone using this particular client will not be able to migrate their data over to Prometheus using Graphite Exporter unless they implement a custom exporter. Yammer + Graphite used to be a popular combination. Perhaps such clients should be supported for mapping using Graphite Exporter.